tests/acceptance/cli_test.py

```diff
@@ -94,6 +95,32 @@ def __str__(self):
         return "waiting for container to be %s" % state
 
 
+class NumberOfLinesCondition(object):
+
+    def __init__(self, proc, expected):
+        self.current_count = 0
+        self.expected = expected
+        self.proc = proc
+        self.lines = ''
+        t = threading.Thread(target=self.read_thread)
+        t.start()
```

Comment:
You shouldn't need any threads here. The Compose binary is run in a subprocess, which can be invoked without blocking.